gamehendge/static/gamehendge/styling.css

```diff
@@ -1,4 +1,10 @@
 #map {
-  height: 600px;
   width: 100%;
- }
\ No newline at end of file
+  height: 100%;
+    padding: 0;
+    position : absolute !important; 
```

Comment:
Do these need to be `important`? I feel like I've heard that `important` is like a manual override and should be avoided when possible because it can complicate things down the like. That being said, I don't know if I'm overzealous about a rule I don't understand.